back-end/prisma/schema.prisma

```diff
@@ -2,7 +2,8 @@
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
 
 generator client {
-  provider = "prisma-client-js"
+  provider      = "prisma-client-js"
+  binaryTargets = ["native", "darwin"]
```

Comment:
Was this file modified by Prisma or by yourself?
If you've modified this file, why you modified it?

Reply:
This file modified referred to the messages on the terminal that `Prisma` showed. I just followed the messages to fix the error.

Reply:
Got it. Thank you for the explanation.